src/models/utils/index.ts

```diff
@@ -19,3 +22,129 @@ export function onlyHasFields(obj: object, fields: string[]): boolean {
 export function isFlagEnabled(Flags: number, checkFlag: number): boolean {
   return (checkFlag & Flags) === checkFlag;
 }
+
+/**
+ * Sets a transaction's flags to its numeric representation
+ *
+ * @param {Transaction} tx A transaction to set its flags to its numeric representation
+ * @returns {void}
+ */
+export function setTransactionFlagsToNumber(tx: Transaction): void {
+  switch (tx.TransactionType) {
+    case 'OfferCreate':
+      tx.Flags = convertOfferCreateFlagsToNumber(tx.Flags as OfferCreateFlags);
+      break;
```

Comment:
maybe `return` instead of `break`. no real reason other than that the matched expression should be the last expression for the function anyway, so i think that that makes it more clear to the reader, who knows there is nothing below the `switch` statement

Reply:
Sure, makes sense.